src/main/kotlin/dev/slne/surf/database/config/redis/RedisConfig.kt

```diff
@@ -0,0 +1,15 @@
+package dev.slne.surf.database.config.redis
+
+import org.spongepowered.configurate.objectmapping.ConfigSerializable
+
+@ConfigSerializable
+data class RedisConfig(
+    val host: String = "localhost",
+    val port: Int = 6379,
+    val password: String? = null,
```

Comment:
```suggestion
    val password: String = "password",
```
so that the config can be generated if there is none